### PR TITLE
Small grammar error in the chainsaw description.

### PR DESCRIPTION
**Pull request by DogZeroX**
That's all.

### CODE REVIEW

File: code/game/objects/items/weapons/tools/saws.dm [hunk 1]
```diff
@@ -69,7 +69,7 @@
 
 /obj/item/weapon/tool/saw/chain
 	name = "chainsaw"
-	desc = "You can cut trees, people walls and zombies with it, just watch out for fuel."
+	desc = "You can cut trees, people, walls and zombies with it, just watch out for fuel."
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/saws.dm]: what about the comma after and

**Comment by NopemanMcHalt** [comment]
image
Upvoting your own PR? _No dignity._

**Comment by DogZeroX** [comment]
@NopemanMcHalt Gay.

File: code/game/objects/items/weapons/tools/saws.dm (hunk 1 above)
Comment (DogZeroX) [reply code/game/objects/items/weapons/tools/saws.dm]: Done. I think.

**Pull request merged**